install/Makefile

```diff
@@ -74,7 +74,7 @@ APACHE_PROGRAM           = /usr/sbin/apache2
 
 # For testing proxying of an external domain, this represents the domain we
 # are proxying from.
-PAGESPEED_TEST_HOST ?= modpagespeed.com
+PAGESPEED_TEST_HOST ?= selfsigned.modpagespeed.com
```

Comment:
Good point.  Hmm, it feels like the variables used by serf_url_async_fetcher_test (which wants selfsigned. and has that compiled in anyway), and used by pagespeed/system/system_tests/map_proxy_domain.sh got conflated.

Maybe the second one can be renamed, and then some of the places it's set can be removed?